### ISSUE TITLE
4.5.1 CTD on end turn 304

### ISSUE THREAD

**Issue by DhuBleidd** [LoneGazebo/Community-Patch-DLL#10704]
_1. Mod Version (X.Y.Z). Current Version: 4.5.1
4.5.1 with EUI

_2. DirectX Version_
DX 10 & 11

_3. List of Other Mods_
R.E.D Modpack
Calypsos Colored Religious Icons
Great Prophet Historical Names
Really Advanced Setup
Reforestation
Units wake up when upgradeable
Wonder race
UI - Improved City View (Vox Populi EUI)

_4. Describe the Issue_
CTD on ending turn 304 no matter what I do on the preceding turns. Game was completely fine up until now - it is the turn of first World Congress vote which is the only new thing AFAIK.
I have also been using the same list of mods for ages with no issues.

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_
[AutoSave_0302 AD-1410.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14499075/AutoSave_0302.AD-1410.zip)

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14499113/Logs.zip)

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_
[CvMiniDump.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/14499127/CvMiniDump.zip)

_8. Steps to reproduce the Issue (Optional)_
Do whatever in the last turn, click next turn on turn 304 and game crashes.

_9. Screenshots of the Issue (Optional)_

Apologies if I forgot something or missed something obvious its my first time submitting here.

**Comment by ilteroi** [comment]
minidump points to CvEspionageAI::BuildOffenseCityList()

```
		TeamTypes eTargetTeam = GET_PLAYER(eTargetPlayer).getTeam();
		CvDiplomacyAI* pTargetDiploAI = GET_PLAYER(eTargetPlayer).GetDiplomacyAI();
```

don't see the problem though - eTargetPlayer is valid?

**Comment by axatin** [comment]
Fixed by 695a227
